modules/tinymce/changelog.txt

```diff
@@ -1,7 +1,9 @@
 Version 5.4.0 (TBD)
     Added Oxide variables for styling the select element and headings in dialog content #TINY-6070
     Added all table menu items to the UI registry, so they can be used by name in other menus #TINY-4866
+    Added additional font related variables for secondary buttons allowing more specific styling. #TINY-6061
```

Comment:
Looks like a merge was resolved incorrectly here, as this is a duplicate of the next line and not something changed in this PR.